### PR TITLE
Anrok Credit notes

### PR DESCRIPTION
**Pull request by stephenlago99**

### CODE REVIEW

File: src/pages/CreditNoteDetails.tsx [hunk 1]
```diff
@@ -1132,3 +1204,8 @@ const InlineLink = styled(Link)`
     text-decoration: none;
   }
 `
+
+const InlineButton = styled.span`
```
Comment (ansmonjol) [inline src/pages/CreditNoteDetails.tsx:1208]: You can use Tailwind classes now! ✨

Comment (ansmonjol) [reply src/pages/CreditNoteDetails.tsx:1208]: Side note, when I had to create such component I used the Link one from react router rom, having a `#` in `to` but the onclick defined! Maybe that can spare you some cutomization here

File: src/components/creditNote/CreditNotesTable.tsx [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+      </div>
+
+      <VoidCreditNoteDialog ref={voidCreditNoteDialogRef} />
+    </ScrollContainer>
+  )
+}
+
+export default CreditNotesTable
+
+const ScrollContainer = styled.div`
```
Comment (keellyp) [inline src/components/creditNote/CreditNotesTable.tsx:379]: Do you think you can merge to tailwind?

File: src/components/customers/CustomerCreditNotesList.tsx [hunk 3]
```diff
@@ -108,14 +107,15 @@ export const CustomerCreditNotesList = ({
...
       ) : (
-        <CreditNotesList
+        <CreditNotesTable
           creditNotes={creditNotes}
           fetchMore={fetchMore}
-          itemClickRedirection={CUSTOMER_CREDIT_NOTE_DETAILS_ROUTE}
-          loading={isLoading}
-          hasSearchQuery={!!variables?.searchTerm}
+          isLoading={isLoading}
+          //hasSearchQuery={!!variables?.searchTerm}
```
Comment (keellyp) [inline src/components/customers/CustomerCreditNotesList.tsx:114]: Is this useful?

File: src/components/creditNote/CreditNotesTable.tsx (hunk 2 above)
Comment (stephenlago99) [reply src/components/creditNote/CreditNotesTable.tsx:379]: That's a good idea, but I say we keep it as a later improvement. Quite a lot of styled components in the file, and migrating them could break things and would require another QA session.

File: src/components/customers/CustomerCreditNotesList.tsx (hunk 3 above)
Comment (stephenlago99) [reply src/components/customers/CustomerCreditNotesList.tsx]: Good point, removed it

File: src/components/creditNote/CreditNotesTable.tsx (hunk 2 above)
Comment (keellyp) [reply src/components/creditNote/CreditNotesTable.tsx:379]: Ok no problem!

**Review by keellyp — APPROVED** [review]

**Pull request merged**